com.unity.netcode.gameobjects/Editor/NetworkManagerHelper.cs

```diff
@@ -65,7 +65,13 @@ private static void ScenesInBuildActiveSceneCheck()
             var scenesList = EditorBuildSettings.scenes.ToList();
             var activeScene = SceneManager.GetActiveScene();
             var isSceneInBuildSettings = scenesList.Count((c) => c.path == activeScene.path) == 1;
+#if UNITY_2023_1_OR_NEWER
```

Comment:
I should we should make it work with the new API after 2023_1 and use the now obsolete API if below 2023_1 and pragma disable warnings/errors. I will reach out to you on Slack later to discuss.